elytron/src/main/java/org/wildfly/extension/elytron/TlsParser.java

```diff
@@ -326,7 +340,7 @@ public void marshallSingleElement(AttributeDefinition attribute, ModelNode mappi
                     .addChild(filteringKeyStoreParser)
             )
             .addChild(keyManagerParser_12_0)
-            .addChild(trustManagerParser)
+            .addChild(trustManagerParser_12_0) // new certificate-revocation-lists attribute
```

Comment:
Since you're adding a new attribute, the existing `tlsParser` variable should first be copy/pasted to a new variable and then that new variable should be updated to reference your new trust manager parser. The Elytron subsystem parser should also be updated to make use of the new TLS parser variable.

Just a heads up that this will also need to be ported to the next Elytron subsystem schema version later.

Reply:
Just a reminder that this part can now be updated.